.travis.yml

```diff
@@ -20,6 +20,8 @@ jobs:
       env: ARCH=ppc64
     - name: "ARCH=ppc64le LD=ld.lld"
       env: ARCH=ppc64le LD=ld.lld
+    - name: "ARCH=riscv LLVM_IAS=1 BOOT=0"
+      env: ARCH=riscv LLVM_IAS=1
```

Comment:
I thought the LLVM_IAS was still in -next?

Reply:
Was merged into mainline before v5.7-rc1.

Reply:
oh, it's in mainline. wow